### DISCUSSION TITLE
seb quit and restart unexpectedly on a window 11 PC

### DISCUSSION THREAD

**Discussion by stephaneferet** [SafeExamBrowser/seb-win-refactoring#795] [General]
Hello, I have a students with a windows 11 machine, when she stats a moodle exam with seb. At the beginning, everything is fine, but after a moment, seb quits and restarts by itself. She still can  do her exam but it's really disturbing. Does anyone already encountered that issue, any lead ?
Thank you and have a great day.
Stephane.

**Comment by dbuechel** [comment]
Can you please provide the log files for an affected session (to be found under `%LocalAppData%\SafeExamBrowser\Logs`) and ideally provide a video or screen recording of the issue itself?
